### DISCUSSION TITLE
type and check

### DISCUSSION THREAD

**Discussion by nycolexavier** [HaxeFlixel/flixel#3064] [Q&A]
I want to create a game and at some point I would have to type and check if the answer is right, how can I do that?

**Comment by Vortex2Oblivion** [comment]
Can you elaborate?

**Comment by DarDevv** [comment]
you can try using `FlxTextInput`, a library created by **starmapo**, you can check answers by using `FlxSignal`, you could use `FlxUIInputText `but i dont recommend you to use it. Here's `FlxTextInput `download link (dont forget to credit starmapo): https://github.com/Starmapo/flixel-text-input
